helm/net-exporter/templates/daemonset.yaml

```diff
@@ -94,9 +94,9 @@ spec:
           # side-effects to measurements.
           requests:
             cpu: 50m
-            memory: 75Mi
+            memory: 150Mi
```

Comment:
Should we actually do this?

Reply:
Eric and I talked about it and didn't see any harm in bumping it until we can verify that the pod list fix actually has enough impact in the busier clusters

Reply:
Opened https://github.com/giantswarm/giantswarm/issues/16797 to remind us to revert it

Reply:
How about only bumping the limits but not the requests? It seems a bit harsh for small clusters.

Reply:
Yeah you're right, that's better. +1 for only bumping the limit

Reply:
+1 Thanks for pointing that out. Made the change.